.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file is here by mistake I assume.

Reply:
My bad. I keep meaning to add a gitignore rule for this.

Reply:
Feel free to open a PR for this.